docs/application/dotnet/guides/nui/text.md

```diff
@@ -244,7 +244,7 @@ To use the text-decoration, set the applicable property:
     label1.Underline = textStyle;
     ```
 
-    By default, the underline height is based on the font metrics. For example, the following text figures are in one pixel height:
+    By default, the underline height is based on the font metrics and the minimum height is one pixel. And the underline color is based on the text color. For example, the following text figures are in one pixel height and the color is the same as the text color:

```

Comment:
Suggest rephrasing to:

By default, the underline height is based on the font metrics and the minimum height is one pixel. The underline color is based on the text color. For example, the following text figures are in one pixel height and the color is the same as the text color:

Please verify

Reply:
@surbhig12 Thanks for rewriting.

@JoogabYun Would you please check @surbhig12's comment and apply it?